superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx

```diff
@@ -314,7 +311,7 @@ class HeaderActionsDropdown extends React.PureComponent {
           <Menu.Item key={MENU_KEYS.EDIT_CSS}>
             <CssEditor
               triggerNode={<span>{t('Edit CSS')}</span>}
-              initialCss={this.state.css}
+              initialCss={customCss}
```

Comment:
@prosdev0107 did you check how `initialCss` works? 
I believe `initialCss` is required in mounting only, not in every rendering. If we use props value directly, `intialCss` will be defined every rendering, which is not its purpose. 

What about the following?
```
componentDidMount() {
   this.setState({
       css: this.props.customCss
   });
}
...
...
<CssEditor
    initialCss={this.state.css}
    ...
/>
```
What do you think @diegomedina248 ?

Reply:
I tried to resolve it with componentDidMount but it doesn't work.